docs/cni/rkt/README.md

```diff
@@ -71,32 +71,29 @@ You should also verify each host can access etcd.  The following will return an
 
     curl -L http://localhost:2379/version
 
-And finally check that Docker is running on both hosts by running
-
-    docker ps
-
 
 ## 2. Starting Calico services
 
-Once you have your cluster up and running, start calico on all the nodes
+Once you have your cluster up and running, start Calico on all the nodes.
+Because `rkt` doesn't support daemonizing containers directly, we'll use [systemd-run][systemd-run]
+to create a systemd service.
 
 On calico-01
 
-    sudo calicoctl node
+    sudo systemd-run --unit=calico-node calicoctl node  --runtime=rkt
 
 On calico-02
 
-    sudo calicoctl node
+    sudo systemd-run --unit=calico-node calicoctl node  --runtime=rkt
 
-This will start a container on each host. Check they are running
+You can check that it's running under `rkt`
 
-    docker ps
+	$ sudo rkt list
+	UUID            APP     IMAGE NAME                              STATE   NETWORKS
+	bc13af40        node    registry-1.docker.io/calico/node:latest running
 
-You should see output like this on each node
 
-    vagrant@calico-01:~$ docker ps
-    CONTAINER ID        IMAGE                           COMMAND                  CREATED             STATUS              PORTS               NAMES
-    ffe6cb403e9b        calico/node:latest              "/sbin/my_init"          21 seconds ago      Up 20 seconds                           calico-node
+You can the logs of the running Calico components using `journalctl -u calico-node`
 
 ## 3. Create the networks
 
```

Comment:
The line below this doesn't english well, get rid of the "and". Try:

"You can configure multiple networks when using rkt. Each is represented by a configuration file..."